.gitea/workflows/publish.yml

```diff
@@ -37,7 +37,7 @@ jobs:
         run: |
           cp ${{ steps.build.outputs.package-file }} ./laconic-so
       - name: "Create release"
-        uses: cerc-io/action-gh-release@gitea-v1
+        uses: actions/action-gh-release@gitea-v1
```

Comment:
This one could be problematic because I had to make code changes in our fork to work with GT. Also that branch isn't present in the GH repo.

Reply:
It seems we could use "self" in that case, but we'd need to migrate the repo to git.vdb.to, because it won't look to an external URL other than Github.

Reply:
We might also be able to specify the complete URL explicitly: https://gitea.com/cerc-io/action-gh-release